mario_1_1.c

```diff
@@ -0,0 +1,41 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+const char HASH = '#';
+const char WHITE = ' ';
+int n;
+    do
+    {
+        n = get_int("Height: ");
```

Comment:
You should add a hint that indicates valid entries.

```suggestion
        n = get_int("Height: (between 1 and 8)");
```